lib/workers/repository/process/lookup/abandonment.ts

```diff
@@ -0,0 +1,32 @@
+import { DateTime } from 'luxon';
+import type { ReleaseResult } from '../../../../modules/datasource/types';
+import { toMs } from '../../../../util/pretty-time';
+import type { LookupUpdateConfig } from './types';
+
+export function calculateAbandonment(
+  releaseResult: ReleaseResult,
+  config: LookupUpdateConfig,
+): ReleaseResult {
+  const { abandonmentThreshold } = config;
+  if (!abandonmentThreshold) {
+    return releaseResult;
+  }
+
+  const abandonmentThresholdMs = toMs(abandonmentThreshold);
+  if (!abandonmentThresholdMs) {
```

Comment:
is there a warning /error somewhere about invalid values?

Reply:
The error is logged in the `toMs` exception handler

Reply:
But added trace anyway